x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -58,6 +58,7 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
         injectReferences: (params, savedObjectReferences) =>
           injectReferences({ logger, params, savedObjectReferences }),
       },
+      autoRecoverAlerts: false,
```

Comment:
I think we can add this in when we switch detection rules to use alert summaries.